### ISSUE TITLE
Why loss without regularization?

### ISSUE THREAD

**Issue by hoonkai** [omni-us/squeezedet-keras#17]
Hi

I noticed that there's a metric for "loss without regularisation", but is it actually meaningful? Also, the implementation of `loss_without_regularization()` appears to be identical to `loss()`. Am I missing something?

Thanks

**Comment by skogsbrus** [comment]
They are identical, yes, but gradient clipping is performed on `loss()` whereas `loss_without_regularization()` is untouched.
